src/member/models.py

```diff
@@ -466,6 +468,7 @@ class Meta:
         'member.Member',
         related_name='+',
         verbose_name=_('billing member'),
+        on_delete=models.CASCADE
```

Comment:
Maybe models.PROTECT? If cascade, when you delete a client's billing member, this client will be deleted.